packages/core/src/components/sections/BannerNewsletter/BannerNewsletter.tsx

```diff
@@ -1,31 +1,39 @@
-import {
-  Banner as UIBanner,
-  BannerContent as UIBannerContent,
-} from '@faststore/ui'
-import Newsletter from '../Newsletter'
+import BannerText, { BannerTextProps } from 'src/components/sections/BannerText'
+import Newsletter, { NewsletterProps } from 'src/components/sections/Newsletter'
 import Section from '../Section'
 import styles from './section.module.scss'
 
-function BannerNewsletter() {
+function BannerNewsletter({
+  banner,
+  newsletter,
+}: {
+  banner: BannerTextProps
+  newsletter: NewsletterProps
+}) {
   return (
     <Section
       className={`${styles.section} section-banner-newsletter layout__content`}
     >
       <div data-fs-banner-newsletter>
-        <UIBanner variant="secondary" colorVariant="light">
```

Comment:
Hey @hellofanny, I think you did some changes here because of the overriding but I don't have enough context, can you help us with this review? 🙌

Reply:
Thanks for tagging me! I think it's ok. We can enable override for this component in another PR :)